egs/vaevc/template/run.sh

```diff
@@ -33,6 +34,12 @@ featsscp="None"
 conf=conf/mlfb_vqvae.yml  # newtork config
 spkr_yml=conf/spkr.yml # speaker config
 
+# synthesis related
+model_step=                     # If not specified, use the latest.
```

Comment:
voc_model_step would be better.

Reply:
The `model_step` here is to specify which conversion model checkpoint to use, not the vocoder.